### PR TITLE
JDK 13 support

### PR DESCRIPTION
**Pull request by tomparle**
JDK 13 support (see https://github.com/playframework/play1/issues/1324 for details on files changed).
New features :
- Play app can be run with JDK 13
- `java.source` parameter can be set up to `12` in application config file. Due to a regression with JDT core lib starting from version 3.19.0, we are currently limited to version 3.18.0 that only supports up to JDK 12.
- lib update : asm 7.2, groovy 2.5.8, javassist 3.26.0-GA, jdt core 3.18.0

Thanks @tazmaniax for his precious contribution !

### CODE REVIEW

File: framework/src/play/mvc/Scope.java [hunk 1]
```diff
@@ -30,7 +30,7 @@
     public static final String COOKIE_EXPIRE = Play.configuration.getProperty("application.session.maxAge");
     public static final boolean SESSION_HTTPONLY = Play.configuration.getProperty("application.session.httpOnly", "false").toLowerCase()
             .equals("true");
-    public static final boolean SESSION_SEND_ONLY_IF_CHANGED = Play.configuration
+    public static boolean SESSION_SEND_ONLY_IF_CHANGED = Play.configuration
```
Comment (tazmaniax) [inline framework/src/play/mvc/Scope.java]: @tomparle is this change related to JDK13?

Comment (tomparle) [reply framework/src/play/mvc/Scope.java]: Yes, see details in my original issue : https://github.com/playframework/play1/issues/1324

Comment (tazmaniax) [reply framework/src/play/mvc/Scope.java]: gotcha, thx!

**Comment by xael-fry** [comment]
@tomparle merge thanks

**Comment by tazmaniax** [comment]
@xael-fry I think my pull request https://github.com/playframework/play1/pull/1338 that you merged shortly before supersedes this one so this merge should be reverted
